src/metabase/analyze/fingerprint/insights.clj

```diff
@@ -19,17 +19,19 @@
 (set! *warn-on-reflection* true)
 
 (defn- last-2 []
-  (fn
-    ([] [])
-    ([acc]
-     (let [cnt (count acc)]
-       (cond (= cnt 0) [nil nil]
-             (= cnt 1) [nil (nth acc 0)]
-             :else acc)))
-    ([acc x]
-     (if (< (count acc) 2)
-       (conj acc x)
-       [(nth acc 1) x]))))
+  (let [null (Object.)]
```

Comment:
nit: I think `none` would be a better name for this, to avoid punning on a null reference.